### PR TITLE
Add permissions test and fix missing permissions

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
   namespace :insights_cloud do
     resources :tasks, only: [:create]
     resource :settings, only: [:show, :update]
-    resources :hits, except: %i[show] do
+    resources :hits, except: [:show, :edit, :destroy, :update, :create, :new] do
```
Comment (Ron-Lavi) [inline config/routes.rb:21]: should we switch to `only` ?

Comment (ShimShtein) [reply config/routes.rb:21]: Didn't try `only` with more definitions inside

**Comment by ShimShtein** [comment]
@Ron-Lavi ready for another round

File: config/routes.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@
   namespace :insights_cloud do
     resources :tasks, only: [:create]
     resource :settings, only: [:show, :update]
-    resources :hits, except: %i[show] do
+    resources :hits, only: [:index, :show] do
```
Comment (Ron-Lavi) [inline config/routes.rb:21]:
```suggestion
    resources :hits, only: [:index] do
```

with `show` included there is an issue in the old host page: `ActionController::ParameterMissing in InsightsCloud::HitsController#show`

**Comment by ShimShtein** [comment]
@Ron-Lavi yup, missed it. Should be OK now

**Review by Ron-Lavi — APPROVED** [review]
Thanks @ShimShtein !

**Pull request merged**